### ISSUE TITLE
en_GB in LangCodes

### ISSUE THREAD

**Issue by jcuenod** [Coral-erm/Coral#49]
I don't have a live install right now - I'm just building an installer. I've just merged in the visual changes by Bradley Droubay and I'm getting

```
Notice: Undefined index: en_GB in /srv/http/Coral/LangCodes.php on line 15
```

Looks like a simple fix - seems it's just missing from an array.

**Comment by bdroubay** [comment]
Sorry all. Because we are translating into both simplified and traditional Chinese, it was necessary to bump the length of the language codes from 2 to 5. If you add a line to the LangCodes.php file that maps en_GB to en_US, and another for the language name, it will resolve the issue. Similar action for French speaking users outside of France. We hope others will begin adding translations as well.

Let me know if you have any further questions. I'm working on merging the other modules.

Thanks,

Brad

**Comment by jeffnm** [comment]
This seems like it is a bug that should be fixed in 2.0.0 if possible.
